server/graphql/v2/object/RequiredBankInformation.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLNonNull, GraphQLString, GraphQLObjectType } from 'graphql';
+import GraphQLJSON from 'graphql-type-json';
+
+import transferwise from '../../../paymentProviders/transferwise';
+
+const RequiredBankInformation = new GraphQLObjectType({
+  name: 'RequiredBankInformation',
+  description: 'A list of required bank information inputs',
+  fields: {
+    fields: {
+      args: {
+        currency: {
+          type: new GraphQLNonNull(GraphQLString),
```

Comment:
Can we express that as a `Currency`? it's already existing as `enum`.

Reply:
We don't have a way to guarantee the subset of currencies this host can transfer to will be the same that we have in the enum.

Reply:
Ok. We could possibly add all standardized currency codes in this GraphQL type:
https://www.iban.com/currency-codes